### PR TITLE
[Database] Build Database GISManager

### PR DESCRIPTION
**Pull request by Jake-wq**
Working on #71

### CODE REVIEW

**Review by brendanfraser597 — APPROVED** [review]

**Review by camrongrant1 — APPROVED** [review]

File: src/main/java/edu/oakland/production/database/DatabaseGisManagerImplementation.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package edu.oakland.production.database;
+
+import edu.oakland.helper.admin.LocationDataPoint;
+
+public class DatabaseGisManagerImplementation {
```
Comment (andrewdimmer) [inline src/main/java/edu/oakland/production/database/DatabaseGisManagerImplementation.java]: This needs to implement the `DatabaseGisManager` Java interface.
```suggestion
public class DatabaseGisManagerImplementation implements DatabaseGisManager {
```

File: src/test/java/edu/oakland/test/database/DatabaseGisManagerTest.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    dgm.passStoreRequest(ldp);
+    assertEquals(ldp, dpss.getLocationDataPoint(0));
+  }
+
+  @Test
+  @DisplayName("Putting in the current satellite returns the new satellite")
+  void currentSatGetsNextSat() {
+    DatabasePersistentStorageStub dpss = new DatabasePersistentStorageStub();
+    DatabaseGisManagerImplementation dgm = new DatabaseGisManagerImplementation(dpss);
+    String currentSat = "";
```
Comment (andrewdimmer) [inline src/test/java/edu/oakland/test/database/DatabaseGisManagerTest.java]: Can we replace this with a non-empty string for more robust testing?

File: src/test/java/edu/oakland/test/database/DatabaseGisManagerTest.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+import edu.oakland.helper.admin.LocationDataPoint;
+import edu.oakland.production.database.DatabaseGisManagerImplementation;
+import edu.oakland.test.database.DatabasePersistentStorageStub;
+import java.lang.IllegalArgumentException;
+import java.time.LocalDateTime;
+import java.util.Random;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("Database GisInterface Test")
```
Comment (andrewdimmer) [inline src/test/java/edu/oakland/test/database/DatabaseGisManagerTest.java]: For sake of standardization to make it easier to read and search the logs, can we change this to `DatabaseGisManagerImplementation Unit Tests`? Thank you!
```suggestion
@DisplayName("DatabaseGisManagerImplementation Unit Tests")
```

**Review by andrewdimmer — CHANGES_REQUESTED** [review]
Couple of minor changes, they you're all set to merge!

**Review by andrewdimmer — APPROVED** [review]
Looks good to me! You're all set to merge!

**Pull request merged**